File: nexora/nexora-front/nexora-front-admin/src/api/test-login.js
import { request } from './request';

// 测试登录请求
const testLogin = async () => {
  try {
    // 方式 1: JSON (正确)
    const result = await request.post('/account/login', {
      username: 'admin',
      password: '123456'
    });
    console.log('✅ Login success:', result);
  } catch (error) {
    console.error('❌ Login failed:', error.response?.data || error.message);
  }
};

testLogin();

File: nexora/test_login.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
测试登录 API
"""
import requests
import json

url = "http://localhost:6061/api/account/login"
data = {
    "username": "admin",
    "password": "123456"
}

print(f"POST {url}")
print(f"Data: {json.dumps(data, indent=2)}")
print()

try:
    # 方式 1: form-data (Spring @RequestParam)
    response = requests.post(url, data=data)
    print("=== Form Data ===")
    print(f"Status: {response.status_code}")
    print(f"Response: {response.text[:500]}")
    
except Exception as e:
    print(f"Error: {e}")
